### PR TITLE
Htaccess dev

### PR DESCRIPTION
**Pull request by lawalyusuf**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,11 +1,18 @@
 Options -Indexes
 RewriteEngine on
+
```
Comment (olucrowther) [inline .htaccess:3]: Remove this new addition

File: .htaccess [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+# Force HTTPS
+RewriteCond %{HTTPS} off
+RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
 RewriteCond %{HTTP_HOST} ^www\.(.*)$ [NC]
 RewriteRule ^(.*)$ http://%1%{REQUEST_URI} [R=301,QSA,NC,L]
```
Comment (olucrowther) [inline .htaccess:9]: Change the rewrite rule here to https instead of http

File: application/.htaccess [hunk 3]
```diff
@@ -1,6 +1,18 @@
-<IfModule authz_core_module>
-    Require all denied
-</IfModule>
-<IfModule !authz_core_module>
-    Deny from all
-</IfModule>
\ No newline at end of file
+Options -Indexes
```
Comment (olucrowther) [inline application/.htaccess:1]: Remove the whole addition from this file

Comment (lawalyusuf) [reply application/.htaccess:1]: Okay. Doing that now

**Pull request merged**